commands/base_command_extended.ts

```diff
@@ -0,0 +1,64 @@
+import { BaseCommand } from "@adonisjs/core/ace";
+
+/**
+ * Convenience interface to pass object arrays to listing prompts without the needless mapping overhead
+ */
+export interface SupportsListing {
+  list(): string;
+}
+
+export default abstract class BaseCommandExtended extends BaseCommand {
+  // Apparently, Adonis tries to serialize everything derived from BaseCommand, even if it is abstract.
+  // Thus, the name property is required for it to compile.
+  static commandName = "abstract_command";
+  static description = "Calling this command does nothing";
```

Comment:
i wonder if removing the default export (and making it named instead) would solve this issue

Reply:
yeah no, this is an error thrown by adonis 😔